lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -5,18 +5,14 @@
 
 foam.CLASS({
   package: 'org.chromium.webidl',
-  name: 'LocalGitIDLFileDAO',
-  extends: 'foam.dao.ProxyDAO',
+  name: 'LocalGitRunner',
+  extends: 'foam.box.Runnable',
```

Comment:
I don't think we want to extend this, right? This is supposed to have two different output boxes which breaks `Runnable`'s structural interface. I thought we were going to manually wire this up to two pipeline entry points (since it's not *really* a `Runnable`). Am I misremembering our offline convo?